server/lomas_server/administration/dashboard/pages/02_database_administration.py

```diff
@@ -3,7 +3,7 @@
 import yaml
 
 from admin_database.utils import get_mongodb
-from administration.mongodb_admin import (
+from mongodb_admin import (
```

Comment:
i think it depends which tab is opened first.. if it is 'about.py' then it is fine like this but I guess you opened on one of the other tabs. Not sure what is the best to do.

Reply:
it should be expected to start from about.py, right ?

Reply:
Yes it should start from about.py. I just moved the mongodb_admin in another level of the directory. (outside the administration subdirectory)